setup.cfg

```diff
@@ -20,8 +20,8 @@ django_settings_module = "IOProject.settings"
 ignore_errors = True
 
 [flake8]
-exclude = ./db/migrations,./venv/
-max-line-length = 79
+exclude = ./db/migrations,./venv/,./*/migrations/*
```

Comment:
max-line-lenght is 79 xd don't change that. Also exclude is looking for pattern so if you add */migration/* it's enough (you can delete ./db/migrations)